### PR TITLE
[fairwinds-insights] Updated insights postgres with partman

### PR DESCRIPTION
**Pull request by jdesouza**
**Why This PR?**
_a short description of why this PR is needed_

Fixes #

**Changes**
Changes proposed in this pull request:

*
*

**Checklist:**

* [x] I have included the name of the chart in the title of this PR in square brackets i.e. `[stable/goldilocks]`.
* [x] I have updated the chart version in `Chart.yaml` following Semantic Versioning.
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] Any new values have been added to the README for the Chart, or `helm-docs --sort-values-order=file` has been run for the charts that support it.

### CODE REVIEW

File: stable/fairwinds-insights/values.yaml [hunk 1]
```diff
@@ -437,7 +437,8 @@ postgresql:
   # -- Set to `true` to run migrations after the upgrade
   postMigrate: false
   image:
-    tag: "14.2.0-debian-10-r94"
+    repository: https://quay.io/fairwinds/postgres-partman
```
Comment (rbren) [inline stable/fairwinds-insights/values.yaml]:
```suggestion
    repository: quay.io/fairwinds/postgres-partman
```

File: stable/fairwinds-insights/values.yaml [hunk 2]
```diff
@@ -437,7 +437,8 @@ postgresql:
   # -- Set to `true` to run migrations after the upgrade
   postMigrate: false
   image:
-    tag: "14.2.0-debian-10-r94"
+    repository: quay.io/fairwinds/postgres-partman
+    tag: 14.4
```
Comment (rbren) [inline stable/fairwinds-insights/values.yaml]:
```suggestion
    tag: "14.4"
```

File: stable/fairwinds-insights/README.md [hunk 3]
```diff
@@ -126,7 +126,9 @@ See [insights.docs.fairwinds.com](https://insights.docs.fairwinds.com/technical-
 | ingress.separate | bool | `false` | Create different Ingress objects for the API and dashboard - this allows them to have different annotations |
 | ingress.extraPaths | object | `{}` | Adds additional path ie. Redirect path for ALB |
 | postgresql.postMigrate | bool | `false` | Set to `true` to run migrations after the upgrade |
-| postgresql.image.tag | string | `"14.2.0-debian-10-r94"` |  |
+| postgresql.image.registry | string | `"quay.io"` |  |
+| postgresql.image.repository | string | `"fairwinds/postgres-partman"` |  |
```
Comment (vitorvezani) [inline stable/fairwinds-insights/README.md:130]: As we might, in the future, add more extensions to this image (`pg_cron`)... I wonder if we could use `postgres-libraries` or some more generic naming for it

Comment (jdesouza) [reply stable/fairwinds-insights/README.md:130]: Right, we can refactor it, when we add pg_cron

**Review by vitorvezani — APPROVED** [review]

**Pull request merged**